addons/stock/tests/test_stock_return_picking.py

```diff
@@ -204,7 +204,7 @@ def test_stock_return_for_exchange(self):
             active_id=original_picking.id, active_ids=original_picking.ids, active_model='stock.picking'
         ).create({})
         # Change the quantity of the product return move from 0 to 1
```

Comment:
You can update the comment
`# Change the quantity of the product return move from 0 to 3`